### DISCUSSION TITLE
Split "Raise wrist" wake up setting in to left-handed and right-handed specific settings

### DISCUSSION THREAD

**Discussion by dennis1248** [InfiniTimeOrg/InfiniTime#1628] [Ideas]
Currently any rapid motion with a slight turn followed by an abrupt stop will trigger a wake-up.

To reduce the amount of false wake ups have a "Raise wrist" setting specifically for left and right handed users. These settings should both take in to consideration the expected rotation and direction of movement of the user's arm.

**Comment by MFAshby** [comment]
Related: https://github.com/JF002/InfiniTime/pull/98

The above PR looks to add a left/right handed setting, so when that's finished and merged it should also be possible for the wakeup code to examine that setting and adjust.
